ui-tests/cypress/support/restAPI/FileBrowserController.js

```diff
@@ -10,8 +10,41 @@ Cypress.Commands.add('fileBrowserController', () => {
 
 export default class FileBrowserController extends AbstractController {
 
+  createFolder(path) {
+    return this.request({
+      url: `${this.apiURL}/directory`,
+      method: 'POST',
+      body: {protectedFolder: false, path: `/${path}`}
+    })
+  }
+
+  deleteFolder(path) {
+    return this.request({
+      url: `${this.apiURL}/directory`,
+      method: 'DELETE',
+      qs: {
+        protectedFolder: false,
+        currentPath: path
+      }
+    });
+  }
+
+  createFile(fileInfo) {
```

Comment:
There's an _uploadFixture_ method in the _abstractController_; check if you can re-use/adapt that method to perform this upload, or, at least, move the logic to process the the fixture file in it

Reply:
Created a new uploadTextFile method in abstractController with the logic to process the fixture file in it